scrapyd_k8s/logging_config.py

```diff
@@ -0,0 +1,13 @@
+import logging
+import sys
+
+def setup_logging():
+    logging.basicConfig(
+        level=logging.DEBUG,
+        format='%(asctime)s %(name)s [%(levelname)s]: %(message)s',
+        handlers=[
+            logging.StreamHandler(sys.stdout)
+        ]
+    )
+
+setup_logging()
```

Comment:
Uhm, shall we call this in main nevertheless?

Also, please provide a way to configure the logging level. Debug is not a good default, remember, this is meant to be production-ready code (even if the joblogs feature may not work everywhere yet).

Reply:
Would you like to have on logger instance for the whole project or have different loggers for each feature/part of the project? I can create an instance right in that config file and then import that instance everywhere (this is what we've done on previous project, it was not too big), but sometimes people create multiple logger instances for each logical part. Depending on it I can refactor the code.

Adding the configurable level.

PS the logger configuration is now called upon importing so we don't have to call it again.

Reply:
Well, let's start with a global log level config for the whole project. Then later we might a way to configure this in more detail.

Ok, I'll leave it to you how to configure the log level. It's just, that in re-usable code, I would call the setup in an init part, so that you can import parts of the code and setup your own logging config. What is logged with what level is the responsibility of the code, but the log level and output format, the responsibility of the main program using (parts of) the code.

But let's try to have some speed, so whatever you think makes sense.